map/framework.cpp

```diff
@@ -2489,15 +2489,14 @@ void Framework::SetRouterImpl(RouterType type)
       return m_model.GetIndex().GetMwmIdByCountryFile(CountryFile(countryFile)).IsAlive();
     };
 
-    //    @TODO(bykoianko, dobriy-eeh). Pass numMwmIds to router and CrossMwmIndexGraph.
-    //    shared_ptr<routing::NumMwmIds> numMwmIds = make_shared<routing::NumMwmIds>();
-    //    m_storage.ForEachCountryFile([&](platform::CountryFile const & file){
-    //      numMwmIds->RegisterFile(file);
-    //    });
+    auto numMwmIds = make_shared<routing::NumMwmIds>();
+    m_storage.ForEachCountryFile([&](platform::CountryFile const & file){
```

Comment:
But even access through raw pointer is worse than direct access, isn't it? In any case, I'm not insisting on how this block of code should be implemented - it's up to you.